core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -85,6 +87,8 @@ public class TableWriter implements TableWriterAPI, MetadataService, Closeable {
     // 3, oldPartitionSize
     // 4, flags (partitionMutates INT, isLastWrittenPartition INT)
     // 5. o3SplitPartitionSize size of "split" partition, new partition that branches out of the old one
+    // 6. ...
```

Comment:
Please add a comment about what is stored at offset 6 if it's used at all

Reply:
done